banyand/measure/aggregate/aggregate_function.go

```diff
@@ -54,6 +54,12 @@ type Function[A, B Input, R Output] interface {
 
 	// Result gives the result for the aggregation.
 	Result() R
+
+	// FirstCumulation gives the first cumulative state for the aggregation.
+	FirstCumulation() A
+
+	// SecondCumulation gives the second cumulative state for the aggregation.
+	SecondCumulation() B
```

Comment:
I think `Cumulation` is strange. Where do you find the naming?

Reply:
Maybe like `cumulative` ?
I used this to represent the intermediate state hold by the struct, like Min.minimum.

Reply:
I am not sure about the point. But Culculation is not a popular word(I am even not sure, is it a real word).
Meanwhile, in a statistic, these things are called factor or parameter of this aggregates.

So, my questions are about the source of this naming?

@hanahmily Could you recheck this?

Reply:
Why not return them through "Result() (A, B, R)"?

Reply:
So "Result() (A, B, R)" is enough. We don't have any chance to get them independently.

Reply:
> I am quite sure, this is a wrong naming.

Yes this naming needs to be corrected. The NewRelic's link only tries to guess the source of "cumulative".

Reply:
Fine, I'll change the interface to `Result() (A, B, R)`, and then once calling Result  returns all we need.